iOS Example Frame/iOS Example Frame/Style/CustomStyle1.swift

```diff
@@ -201,7 +201,7 @@ private struct StyleOrganiser {
         var textAlignment: NSTextAlignment = .natural
         var text: String = ""
         var isSupportingNumericKeyboard: Bool = true
-        var height: Double = 30
+        var height: Double = 56
```

Comment:
Any reason to change this ?

Reply:
Yes, there is a conflict in the current UI creation where it sometimes uses this constraint, but other times a default fallback value.

Should be raised as well at some point but thought I'd save you guys the horrors 😄

Reply:
Was this broken with 30 and now changing to 56 to follow UI designs ?

Reply:
Sometimes the default from the fallback is applied, other times the style provided one is applied.

In practice, this leads to some of the debug warnings of unsatisfied constraints you see in Xcode when you run the app. This number specifically made the PhoneNumberInput become 30 pixels height on refresh (other components don't trigger that refresh so they wouldn't have shown...

Reply:
Unsatisfied constraints warnings, would be nice to fix at some point...

<img width="1400" alt="Screenshot 2022-09-07 at 16 31 39" src="https://user-images.githubusercontent.com/102028170/188918726-ffd7bc9c-2dc7-4212-b81d-d974fc74b1f0.png">